src/shared/components/table/styles/table.module.scss

```diff
@@ -10,12 +20,13 @@
   margin: 1em 0;
   border-spacing: 0;
   border-block-end: 1px solid #c2c4c4;
+  border-collapse: collapse;
 
   & th {
```

Comment:
In the current version, the table header is sticky. 

  & thead {
    position: sticky;
    top: 0;
    z-index: 1;
  }
  
With the border-collapse: separate - it looks neat if we have it sticky.